examples/asr/conf/speech_to_text_hf_finetune.yaml

```diff
@@ -0,0 +1,175 @@
+name: "Speech_To_Text_HF_Finetuning"
+
+# use `init_from_nemo_model` or `init_from_pretrained_model` to initialize the model
+# We do not currently support `init_from_ptl_ckpt` to create a single script for all types of models.
+init_from_nemo_model: null # path to nemo model
+init_from_pretrained_model: null  # name of pretrained NeMo model, e.g., `stt_en_fastconformer_transducer_large`
+
+model:
+  sample_rate: 16000
+  compute_eval_loss: false # eval samples can be very long and exhaust memory. Disable computation of transducer loss during validation/testing with this flag.
+  log_prediction: true # enables logging sample predictions in the output during training
+  rnnt_reduction: 'mean_volume'
+  skip_nan_grad: false
+
+  # configs for huggingface load_dataset function
+  data_path: "librispeech_asr"
+  data_name: null
+  num_proc: 8  # Number of processes when downloading and generating the dataset locally.
+  streaming: false # set True to use streaming mode, which doesn't wait for data downloading but each training step takes longer in the first epoch. If True, you'll need to specify trainer.max_steps instead of trainer.max_epochs.
+
+  # keys for audio, sample_rate and text in the huggingface dataset, keys seperated by `.` for nested fields.
+  audio_key: "audio.array"
+  sample_rate_key: "audio.sampling_rate"
+  text_key: "text"
+
+  # simple text cleaning, by default converts all chars to lower-case and only keeps alpha-numeric chars.
+  normalize_text: true
+  symbols_to_keep: "'"
+
+  train_ds:
+    normalize_text: ${model.normalize_text}
+    symbols_to_keep: ${model.symbols_to_keep}
+    manifest_filepath: "hugginface"  # set to a not None value to avoid breaking existing code
+    audio_key: ${model.audio_key}
+    sample_rate_key: ${model.sample_rate_key}
+    text_key:  ${model.text_key}
+    hf_data_cfg:  # configs for huggingface load_dataset, add more if needed
+      path: ${model.data_path}
+      name: ${model.data_name}
+      split: 'validation.clean'
```

Comment:
Why does train ds have split validation.clean?

Reply:
oh it's just for debugging, I'll change it to train. Also, I'll add support for multiple HF datasets in training, currently only one is allowed